### PR TITLE
Feat/mn-256/update damage pop up

### PR DESCRIPTION
**Pull request by rohitsharma120582**
Implementing the small popup that pops up when the user wants to add or edit a damage on the car.

## Technical description
- Create a component called `UpdateDamagePopUp`
- Add animation to show the component from bottom
- Add gesture on small-bar to slide the popup from "minimal" to "full" and vice versa
- Integrating `DamageManipulator` component

## UI

<p>
Screenshot 2023-04-27 at 4 15 37 PM

Screenshot 2023-04-27 at 4 15 49 PM

Screenshot 2023-04-27 at 4 15 58 PM

Screenshot 2023-04-27 at 4 16 13 PM
</p>

### CODE REVIEW

File: packages/inspection-report/src/components/DamageReport/DamageReport.js [hunk 1]
```diff
@@ -53,16 +56,26 @@ const styles = StyleSheet.create({
 function DamageReport() {
   const { t } = useTranslation();
   const [tabIndex, setTabIndex] = useState(0);
+  const [isPopUpVisible, setIsPopUpVisible] = useState(false);
 
   const onTabHandler = useCallback((index) => setTabIndex(index), []);
 
+  const onPopUpDismiss = useCallback(() => {
+    console.log('Popup Dismiss...');
```
Comment (souyahia-monk) [inline packages/inspection-report/src/components/DamageReport/DamageReport.js:64]: Remove this `console.log`

File: packages/inspection-report/src/components/DamageReport/DamageReport.js [hunk 2]
```diff
@@ -53,16 +56,26 @@ const styles = StyleSheet.create({
...
+    setIsPopUpVisible(0);
```
Comment (souyahia-monk) [inline packages/inspection-report/src/components/DamageReport/DamageReport.js:65]: Replace by `setIsPopUpVisible(false)`

File: packages/inspection-report/src/components/DamageReport/DamageReport.js (hunk 1 above)
Comment (rohitsharma120582) [reply packages/inspection-report/src/components/DamageReport/DamageReport.js]: Done

File: packages/inspection-report/src/components/DamageReport/DamageReport.js (hunk 2 above)
Comment (rohitsharma120582) [reply packages/inspection-report/src/components/DamageReport/DamageReport.js]: Done

File: packages/inspection-report/src/components/DamageReport/DamageReport.js [hunk 3]
```diff
@@ -82,10 +93,28 @@ function DamageReport() {
...
+            }
           </Text>
         </View>
       </View>
+      {isPopUpVisible ? (
+        <UpdateDamagePopUp
+          part="wheel_front_right"
+          damage={null}
+          damageMode="all"
+          imageCount={3}
```
Comment (deepakglobant) [inline packages/inspection-report/src/components/DamageReport/DamageReport.js:113]: Can we use a const variable here rather than using the magic number?

**Review by souyahia-monk — APPROVED** [review]

**Pull request merged**